app/uk/gov/hmrc/gform/testonly/SavedForm.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.testonly
+
+import java.time.Instant
+import julienrf.json.derived
+import play.api.libs.functional.syntax._
+import play.api.libs.json.OFormat
+import play.api.libs.json.Reads._
+import play.api.libs.json._
+
+case class SavedForm(
+  templateId: String,
+  formId: String,
+  userId: String,
+  savedAt: Instant
+)
```

Comment:
Let call this `Snaphost`. 

Why this stores `formId` and `userId`?

It should store the raw data in something like `Map[String, String]`. Perhaps we can reuse `FormData`.